client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -15,21 +41,123 @@
 	text-align: center;
 }
 
+.design-selector__grid-container {
+	position: absolute;
+	margin-top: 1em;
+
+	&.exit,
+	&.exit-done {
+		overflow-y: hidden;
+		height: 100vh;
+
+		.design-selector__grid {
+			@include design-selector-hide();
+		}
+	}
+}
 .design-selector__grid {
+	@include design-selector-transition();
 	display: grid;
 	grid-gap: 4.5em;
 
 	@include breakpoint( '>660px' ) {
 		// stylelint-disable unit-whitelist
-		grid-template-columns: repeat( 2, 1fr );
+		grid-template-columns: 1fr 1fr;
 		// stylelint-enable unit-whitelist
 	}
+
+	@include design-selector-show();
 }
 
-.design-selector__design-option {
-	transition: transform 100ms ease-in-out;
+.page-selector__grid-container {
+	position: absolute;
+	top: $design-selector-header-vertical-shift + $deisgn-selector-selection-space;
+
+	&,
+	&.enter,
+	&.exit-active.exit-active, // duplicate selector to override `.exit` below
+	&.exit-done {
+		.page-layout-selector {
+			@include design-selector-hide();
+		}
+	}
+
+	&.exit,
+	&.enter-active,
+	&.enter-done {
+		.page-layout-selector {
+			@include design-selector-show();
+			transform: translate3d( 0, 0, 0 );
+		}
+	}
+}
+.page-layout-selector {
+	@include design-selector-transition();
+	background: $white;
+	box-shadow: gray 0 0 12px; // use a sass variable
+	padding: 2em 4em;
+}
+
+.design-selector__header-container {
+	height: $design-selector-header-height;
+}
+
+.page-layout-selector__title {
+	font-weight: bold;
+	margin-bottom: 1em;
+	font-size: 1.3em;
+}
+
+.page-layout-selector__grid {
+	display: grid;
+	grid-gap: 0.75em;
+	@include breakpoint( '>660px' ) {
+		// stylelint-disable unit-whitelist
```

Comment:
Nit; `// stylelint-disable-next-line unit-whitelist` instead of two-liner?

Removing the need for these at https://github.com/Automattic/wp-calypso/pull/38547 anyway.